CorsixTH/Lua/entities/humanoid.lua

```diff
@@ -460,69 +460,18 @@ function Humanoid:getCurrentMood()
   end
 end
 
-local function Humanoid_startAction(self)
+--! Start the next (always first) action in the queue.
+function Humanoid:startAction()
```

Comment:
Any perceived issues changing this from local to global?

Reply:
Don't call it outside this file?  :P 

Nothing currently knows about this method, so that should be fine. Not sure what needs fixing if any.